scipy/ndimage/src/ni_measure.c

```diff
@@ -205,7 +205,7 @@ typedef struct {
     npy_intp index;
     void *next, *prev;
     npy_uint16 done;
```

Comment:
The type for done used to be `uint8` instead of `uint16`:
b3a66bf#diff-23265bac697090bca24e16cd0e08f306L663

Maybe this should also be reverted.

Reply:
I reverted the type for `done` (while keeping the order of the fields sorted by decreasing size).